src/nifake/unit_tests/test_session.py

```diff
@@ -837,10 +838,27 @@ def test_return_timedeltas(self):
             assert returned_timedeltas == expected_timedeltas
             self.patched_library_interpreter.return_list_of_durations_in_seconds.assert_called_once_with(len(time_values))
 
+    def test_function_with_int_flag_parameter_with_specific_flag(self):
+        flags = nifake.IntFlagEnum.C
+        self.patched_library_interpreter.function_with_int_flag_parameter.side_effect = None
+        self.patched_library_interpreter.function_with_int_flag_parameter.return_value = None
+        with nifake.Session('dev1') as session:
+            session.function_with_int_flag_parameter(flags)
+        self.patched_library_interpreter.function_with_int_flag_parameter.assert_called_once_with(flags)
+        called_arg = self.patched_library_interpreter.function_with_int_flag_parameter.call_args[0][0]
+        assert called_arg == 1 or called_arg.value == 4
```

Comment:
>         called_arg = self.patched_library_interpreter.function_with_int_flag_parameter.call_args[0][0]
>        assert called_arg == 1 or called_arg.value == 4

I don't understand what you're doing here.
You've alerady asserted that we passsed the right thing down to the library interpreter.

Now you're confirming it again?
And what's going on with the "== 1 or ... == 4"? Why would it equal 1?